dev-python/influxdb/influxdb-5.2.3.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2020 Gentoo Authors
```

Comment:
Please look up on how to write good commit messages.

Reply:
I was under the impression that version bumps in the Gentoo repo typically didn’t say anything about what changed in the underlying package, and that you had to go upstream to find that out. But sure, I’ve added a little bit more text in beea73e70534; let me know if there is anything further you feel is worth putting in the message.

Reply:
But they usually do say what the new version is.